core/templates/dev/head/pages/maintenance-page/maintenance-page.mainpage.html

```diff
@@ -74,7 +74,45 @@ <h2>The Oppia site is temporarily unavailable.</h2>
     <script src="/templates/dev/head/components/forms/RequireIsFloatDirective.js"></script>
     <script src="/templates/dev/head/components/forms/ObjectEditorDirective.js"></script>
     <script src="/templates/dev/head/domain/utilities/UrlInterpolationService.js"></script>
-    <script src="/templates/dev/head/pages/maintenance/Maintenance.js"></script>
+    <script src="/templates/dev/head/pages/maintenance-page/maintenance-page.module.js"></script>
```

Comment:
Did you miss the controller script of this page?

Reply:
i think yes
Done